### ISSUE TITLE
Cannot login anymore, cannot access any games or NBA TV

### ISSUE THREAD

**Issue by nzeppos** [maxgalbu/xbmc.plugin.video.nba#18]
Last used the latest version of the addon (0.8.2) about a couple of weeks ago and it was working perfectly fine. Renewed my LP subscription, tried again today and nothing seems to be working (eg. cannot watch any archived games or NBA TV), as all requests fail with a login error. Credentials work OK on the website.

This must be a new issue, possibly related to 2016-17 League Pass API? (not sure if they've changed something since last year).

Here is the debug log snippet when trying to launch live NBA TV (tokens, usernames etc have been masked):

```
21:17:10 1384684.000000 T:1963282432   DEBUG: Keyboard: scancode: 0x1c, sym: 0x000d, unicode: 0x0000, modifier: 0x0
21:17:10 1384684.125000 T:1963282432   DEBUG: OnKey: return (0xf00d) pressed, action is Select
21:17:10 1384684.125000 T:1963282432   DEBUG: OnPlayMedia plugin://plugin.video.nba/?url=&mode=nbatvlive&name=Watch+live
21:17:10 1384684.125000 T:1963282432   DEBUG: CAnnouncementManager - Announcement: OnAdd from xbmc
21:17:10 1384684.125000 T:1963282432   DEBUG: GOT ANNOUNCEMENT, type: 2, from xbmc, message OnAdd
21:17:10 1384684.250000 T:1963282432   DEBUG: StartScript - calling plugin NBA League Pass('plugin://plugin.video.nba/','210','?url=&mode=nbatvlive&name=Watch+live')
21:17:10 1384684.250000 T:1424208800   DEBUG: Thread LanguageInvoker start, auto delete: false
21:17:10 1384684.250000 T:1963282432   DEBUG: WaitOnScriptResult - waiting on the NBA League Pass (id=594) plugin...
21:17:10 1384684.250000 T:1424208800    INFO: initializing python engine.
21:17:10 1384684.250000 T:1424208800   DEBUG: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): start processing
21:17:10 1384684.375000 T:1424208800   DEBUG: -->Python Interpreter Initialized<--
21:17:10 1384684.375000 T:1424208800   DEBUG: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): the source file to load is "/storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py"
21:17:10 1384684.375000 T:1424208800   DEBUG: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): setting the Python path to /storage/.kodi/addons/xbmc.plugin.video.nba-master:/storage/.kodi/addons/script.common.plugin.cache/lib:/storage/.kodi/addons/script.module.dateutil/lib:/storage/.kodi/addons/script.module.pytz/lib:/storage/.kodi/addons/script.module.six/lib:/usr/lib/python27.zip:/usr/lib/python2.7:/usr/lib/python2.7/plat-linux2:/usr/lib/python2.7/lib-tk:/usr/lib/python2.7/lib-old:/usr/lib/python2.7/lib-dynload:/usr/lib/python2.7/site-packages:/usr/lib/python2.7/site-packages/gtk-2.0
21:17:10 1384684.375000 T:1424208800   DEBUG: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): entering source directory /storage/.kodi/addons/xbmc.plugin.video.nba-master
21:17:10 1384684.375000 T:1424208800   DEBUG: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): instantiating addon using automatically obtained id of "plugin.video.nba" dependent on version 2.20.0 of the xbmc.python api
21:17:11 1384684.875000 T:1424208800    INFO: ##### NBA League Pass: Chosen quality_id 0 and target_video_height 72060
21:17:12 1384685.625000 T:1424208800   DEBUG: ##### NBA League Pass: Login reponse: <?xml version="1.0" encoding="UTF-8"?>
                                            <result>
                                            	
                                            		<code>loginsuccess</code>
                                            	
                                            	
                                            		<data>
                                            			
                                            				
                                            					
                                            					
                                            						<premium><![CDATA[true]]></premium>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<trackUsername><![CDATA[***********]]></trackUsername>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<givenName><![CDATA[****]]></givenName>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<refresh><![CDATA[true]]></refresh>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<userName><![CDATA[*******]]></userName>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<premiumpkg><![CDATA[true]]></premiumpkg>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<hasSubscription><![CDATA[true]]></hasSubscription>
                                            					
                                            				
                                            			
                                            				
                                            					
                                            					
                                            						<live><![CDATA[true]]></live>
                                            					
                                            				
                                            			
                                            		</data>
                                            	
                                            </result>
21:17:12 1384685.750000 T:1424208800   DEBUG: ##### NBA League Pass: nba tv live: the body of publishpoint request is: nt=1&ppid=*******&type=channel&id=0
21:17:12 1384685.750000 T:1963282432   DEBUG: DialogProgress::Open called
21:17:12 1384685.750000 T:1963282432   DEBUG: ------ Window Init (DialogProgress.xml) ------
21:17:13 1384686.500000 T:1424208800   DEBUG: ##### NBA League Pass: nba live tv: failed getting url: http://watch.nba.com/nba/servlets/publishpoint <html><head><title>Apache Tomcat/6.0.36 - Error report</title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}--></style> </head><body><h1>HTTP Status 400 - </h1><HR size="1" noshade="noshade"><p><b>type</b> Status report</p><p><b>message</b> <u></u></p><p><b>description</b> <u>The request sent by the client was syntactically incorrect.</u></p><HR size="1" noshade="noshade"><h3>Apache Tomcat/6.0.36</h3></body></html>
21:17:13 1384686.500000 T:1424208800   DEBUG: POParser: loaded 12 strings from file /storage/.kodi/addons/xbmc.plugin.video.nba-master/resources/language/English/strings.po
21:17:13 1384686.500000 T:1424208800    INFO: CPythonInvoker(594, /storage/.kodi/addons/xbmc.plugin.video.nba-master/leaguepass.py): script successfully run
21:17:13 1384686.500000 T:1963282432   DEBUG: WaitOnScriptResult- plugin returned successfully
21:17:13 1384686.625000 T:1963282432   DEBUG: Loading settings for
21:17:13 1384686.625000 T:1963282432   DEBUG: CPlayerCoreFactory::GetPlayers()
```

Looks like I'm getting a "loginsuccess" response, but immediately afterwards the next request fails with "The request sent by the client was syntactically incorrect".

Could you please look into it? The new season is upon is and this addon is really a must, the ultimate plugin for those watching NBA LP via Kodi.

Thanks.

**Comment by maxgalbu** [comment]
Closing in favor of #17
